zerver/migrations/0067_archived_models.py

```diff
@@ -105,6 +105,7 @@ class Migration(migrations.Migration):
                             "collapsed",
                             "mentioned",
                             "wildcard_mentioned",
+                            "online_mentioned",
```

Comment:
Note also that it doesn't make sense to modify old migrations; the schema migrations docs should explain this, but if not, read about Django's migrations system.  The whole point is that migrations files are static once implemented so that you can apply a series of them to change the database from its original format from a previous version to the modern format.